### PR TITLE
Storybook dump

### PR DESCRIPTION
**Pull request by JonoMercier**
@arizonatribe and @AlexMaskovyak - Could you guys take a look at this and see if we can get this merged into development?
I changed the naming scheme to match what we did in the internal branded-components and I adjusted the index to match.

### CODE REVIEW

**Comment by AlexMaskovyak** [comment]
@JonoMercier It looks like your branch may not be up to date with development. Can you re-sync development with your branch to remove these conflicts?

File: .stories/common/complete-form.stories.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        <CenterDiv>
+            <StoryComponent />
+        </CenterDiv>
+    </ThemeProvider>
+    ))
+
+.add('Sample Field',
+    compose(
+    withInfo('An input element type of text box.'),
+    withNotes('An input element type of text box.')
```
Comment (AlexMaskovyak) [inline .stories/common/complete-form.stories.js]: What's the difference of withInfo and withNotes? I notice that they're identical.

Comment (TimGraf) [reply .stories/common/complete-form.stories.js]: Notes is a different add on.

https://github.com/storybooks/storybook/tree/master/addons/notes

File: package.json [hunk 2]
```diff
@@ -86,7 +88,7 @@
     "@storybook/addon-knobs": "^3.2.15",
     "@storybook/addon-links": "^3.2.15",
     "@storybook/addon-notes": "^3.2.15",
-    "@storybook/addon-storyshots": "^3.2.15",
+    "@storybook/addon-storyshots": "^3.2.16",
     "@storybook/react": "^3.2.13",
     "babel-cli": "6.24.1",
     "babel-eslint": "7.2.3",
```
Comment (TimGraf) [inline package.json]: @arizonatribe Do you have a recommended convention for dependency versions?  

```
version Must match version exactly
>version Must be greater than version
>=version etc
<version
<=version
~version "Approximately equivalent to version" See semver
^version "Compatible with version" See semver
1.2.x 1.2.0, 1.2.1, etc., but not 1.3.0
http://... See 'URLs as Dependencies' below
* Matches any version
"" (just an empty string) Same as *
version1 - version2 Same as >=version1 <=version2.
range1 || range2 Passes if either range1 or range2 are satisfied.
git... See 'Git URLs as Dependencies' below
user/repo See 'GitHub URLs' below
tag A specific version tagged and published as tag See npm-dist-tag
path/path/path See Local Paths below
```

Should we always use `^` with the version, or maybe always allow a new patch version number, `*`, in the last digit?

File: .stories/common/complete-form.stories.js (hunk 1 above)
Comment (JonoMercier) [reply .stories/common/complete-form.stories.js]: Yes.  While talking with @arizonatribe and Scott, we discussed that the two add ons can serve different purposes.  The "Notes" add on actually shows a note right in the same panel that the user sees when they engage the component (like with knobs).  The "With info" is more of a developer tool-kit, and shows sample code along with any other notes about the element.

So we can use one for "forward facing" descriptions, and the other with more technical explanations (if needed).

**Comment by JonoMercier** [comment]
Trying to pull from dev and fix conflicts.  Also there is an issue with how I named them, it's actually 
<element.story.js> and I had <element.stories.js> so I corrected that.

File: .stories/common/complete-form.stories.js (hunk 1 above)
Comment (AlexMaskovyak) [reply .stories/common/complete-form.stories.js]: Okay, I suppose I'm fine with this for now.

**Pull request merged**